log_outgoing_requests/config_reset.py

```diff
@@ -4,8 +4,10 @@
 from .tasks import reset_config
 
 
-def schedule_config_reset():
-    reset_after = settings.LOG_OUTGOING_REQUESTS_RESET_DB_SAVE_AFTER
+def schedule_config_reset(config):
```

Comment:
not sure how I can add the `OutgoingRequestsLogConfig` typehint here without the docs check failing, I tried it with `if TYPE_CHECKING` but that didn't work

Reply:
`from __future__ import annotations` and then a conditional import tends to work usually, but instead, why not pass the `self.reset_db_save_after` so you can provide a typehint `reset_after: int | None` so you're only working in primitives :)